icu4j/main/common_tests/src/test/java/com/ibm/icu/dev/test/format/NumberFormatTest.java

```diff
@@ -7120,4 +7120,63 @@ public TestData(String unitIdentifier, Integer inputValue, UnitWidth width, ULoc
         }
 
     }
+
+    @Test
+    public void TestPortionFormat() {
+        MeasureUnit unit = MeasureUnit.forIdentifier("portion-per-1e9");
+        LocalizedNumberFormatter formatter = NumberFormatter.withLocale(ULocale.ENGLISH).unit(unit)
+                .unitWidth(UnitWidth.FULL_NAME);
+        String formatted1 = formatter.format(1).toString();
+        assertEquals("1 portion per 1e9", "1 part per billion", formatted1);
+
+        String formatted2 = formatter.format(2).toString();
+        assertEquals("1000 portion per 1e9", "2 parts per billion", formatted2);
+
+        String formatted3 = formatter.format(1000000).toString();
+        assertEquals("1000000 portion per 1e9", "1,000,000 parts per billion", formatted3);
+
+        String formatted4 = formatter.format(1000000000).toString();
+        assertEquals("1000000000 portion per 1e9", "1,000,000,000 parts per billion", formatted4);
```

Comment:
These four test cases all seem to be doing the same thing.  In all four of them, the "unit" part of the string is exactly the same.  Why do you need all four, and is there _other_ testing you should be doing where the "unit" part of the string _does_ change?  (Okay, I'm wrong: _one_ of these _is_ different and the other _three_ are all the same...)

And if all you're doing here is making sure that "portion-per-1e9" comes out properly localized (in English) as "parts per billion", isn't there a more general test you could just add this too?